packages/loopring_v3/package.json

```diff
@@ -26,22 +26,23 @@
   },
   "scripts": {
     "watch": "npm-watch compile",
-    "transpile": "rm -rf ./transpiled; copyfiles ./build/**/* ./transpiled && tsc && ./genAbi.sh",
+    "transpile": "rm -rf ./transpiled; copyfiles ./build/**/* ./transpiled && tsc && npm run genabi",
     "test": "npm run transpile && node --max-old-space-size=4096 `which truffle` test --network development",
     "docker": "docker-compose up --build --abort-on-container-exit; docker-compose logs -f test",
     "testdocker": "npm run transpile && truffle test --network docker",
     "compile": "truffle compile",
-    "build": "./install && make && yarn run compile",
+    "build": "cd ../../../protocol3-circuits/ && git pull && cd - && ./install && make && npm run compile",
```

Comment:
Maybe we can also remove the `keys` folder here (or maybe in the makefile, not sure what makes the most sense). Because everybody sometimes forgets doing that (me included).